src/VisualStudio/VisualBasic/Impl/LanguageService/VisualBasicPackage.vb

```diff
@@ -23,18 +23,18 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic
     ' The option page configuration is duplicated in PackageRegistration.pkgdef.
```

Comment:
Running VS through VS with these changes doesn't show the updated name.

I believe I need to make some changes in `PackageRegistration.pkgdef`, but I don't quite understand how this works.

Reply:
I think you may have to update some resource strings in this file, and then rebuild: https://github.com/dotnet/roslyn/blob/bad1590b5c206aef9853c1518c7ebe354c0585c2/src/VisualStudio/VisualBasic/Impl/VSPackage.resx#L138

Reply:
Those resource strings get picked up by the pkgdef here: https://github.com/dotnet/roslyn/blob/bad1590b5c206aef9853c1518c7ebe354c0585c2/src/VisualStudio/VisualBasic/Impl/PackageRegistration.pkgdef#L48

Reply:
@allisonchou Thanks!
Can you confirm that the resource "103" used *only* for UI display? In case it's used for persisting user options, etc, then the change won't be desired.

Reply:
I'm actually still seeing it as "Basic" after changing the resource :(
Will push in case you want to take a look.

Reply:
Hmm that's weird! I'm not really sure what's missing then, although I can try to help take a look in a bit.
For context, I did something similar in Razor a few weeks back where I renamed the Tools->Options setting from 'ASP.NET Core Razor Language Services' to 'Razor (ASP.NET Core)'. If I recall correctly, it was only the resource string that needed to be changed, but maybe VB's settings work differently? Here's the PR for reference - https://github.com/dotnet/aspnetcore-tooling/pull/3317

Reply:
@Youssef1313 I _think_ resources referenced in the registry with the "#" format might be positional (I think they're read by native code), so perhaps try manually editing the resx file and moving "103" back to above "104" might help?

Reply:
Going to delete `%LocalAppData%\Microsoft\VisualStudio\16.0_dfba12b8RoslynDev` and `%LocalAppData%\Microsoft\VisualStudio\16.0_dfba12b8$Client$RoslynDev` and check again now.

Reply:
Same result :confused:

![image](https://user-images.githubusercontent.com/31348972/117949556-865da780-b312-11eb-95de-473da9e577ac.png)

Could it be something outside Roslyn?

Reply:
I think that's controlled by https://github.com/dotnet/roslyn/blob/bad1590b5c206aef9853c1518c7ebe354c0585c2/src/VisualStudio/VisualBasic/Impl/PackageRegistration.pkgdef#L170.

Reply:
And finally!

![image](https://user-images.githubusercontent.com/31348972/118235529-2d6a4c80-b495-11eb-9d64-c2b7ee5417ce.png)

Thanks a ton @jasonmalinowski ❤️